doc/apiary/v2/fiware-ngsiv2-reference.apib

```diff
@@ -282,6 +270,96 @@ Error list (HTTP response code in paranthesis):
+is a geospatial data interchange format based on the JavaScript Object Notation (JSON). GeoJSON provides greater flexibility
+allowing the representation of complex geospatial features.
+
+Additionally, geospatial properties might be declared by means of [civic addresses](https://schema.org/PostalAddress),
+but in that particular case entity providers will have to implement their own geocoding procedures,
+so that geographical queries can be resolved. 
+
+### Simple Location Format
+
+Simple Location Format supports basic geometries (*point* , *line* , *box* , *polygon*) and covers the typical
+use cases when encoding geographical locations. It has been inspired by [GeoRSS](http://www.georss.org/simple.html).
+
+It is noteworthy that the Simple Location Format is not intended to represent complex positions on Earth surface.
+For instance, those applications which require to capture altitude coordinates will have to use GeoJSON as
+representation format for the geospatial properties of their entities. 
+
+A context attribute  which represents a location encoded using the Simple Location Format
+must conform to the following syntax:
+
+* The attribute type must be one of the following values: (`geo:point`, `geo:line`, `geo:box` or `geo:polygon`).
+* The attribute value must be a list of coordinates, using the *WGS84* datum,
+with latitude and longitude units of decimal degrees. Such coordinate list will allow to encode the geometry
+specified by the `type` attribute and will be represented according to the specific rules defined below:
+
+  * Type `geo:point`:   the attribute value must contain a string containing a valid latitude-longitude pair, separated by a comma. 
+  * Type `geo:line`:    the attribute value must contain a string array of valid latitude-longitude pairs. There must be at least two pairs.
+  * Type `geo:polygon`: the attribute value must contain a string array of valid latitude-longitude pairs.
+There must be at least four pairs, with the last being identical to the first (so a polygon has a minimum of three actual points).
+  * Type `geo:box`:     A bounding box is a rectangular region, often used to define the extents of a map or
+a rough area of interest. A box is represented by a two-length string array of latitude-longitude pairs.
+The first pair is the lower corner, the second is the upper corner.
+
+The examples below illustrate the referred syntax:
+
+```
+{
+  "location": {
+    "value": "41.3763726, 2.186447514",
+    "type": "geo:point"
+  }
+}
+```
+
+```
+{
+  "location": {
+    "value": [
+              "40.63913831188419, -8.653321266174316",
+              "40.63881265804603, -8.653149604797363"
+    ],
+    "type": "geo:box"
+  }
+}
+```
+
+### GeoJSON
+
+A context attribute which represents a location encoded using GeoJSON must conform to the following syntax:
+
+* The attribute type must be the MIME type assigned by IETF to the GeoJSON format. Currently 'application/vnd.geo+json'.
```

Comment:
"atribute type" -> "attribute NGSI type" (to avoid the ambiguety that "type" in the case of attribute may refer to NGSI types or JSON native types).